Added the cheatsheet section

ayan-joshi opened the pull request:
## Related Issue
#413 

## Proposed Changes
Added the cheatsheet in the navbar.
when we click the cheasheet it takes us to the section of cheatsheet or the cheatsheets.html file.

## Additional Info
- Any additional information or context

## Checklist

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] ✅ My code follows the code style of this project.
- [ ] 📝 My change requires a change to the documentation.
- [ ] 🎀 I have updated the documentation accordingly.
- [ ] 🌟 ed the repo

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)


<!--- Provide a general summary of your changes in the Title above -->

<!--- If it fixes an open issue, please link to the issue here. -->

<!--- Describe your changes in detail -->

<!--- Why is this change required? What problem does it solve? -->

<!--- see how your change affects other areas of the code, etc. -->



## Output Screenshots
| Screenshot #1      | Screenshot #2  |
| ----------- | ----------- |
| Title goes here  | Title goes here     |
| Image goes here  | Image goes here

ayan-joshi committed:
commit

ayan-joshi commented:
completed it @agamjotsingh18 have a look

agamjotsingh18 commented:
I told to make it in the home page itself instead of shifting to a new page (like services section cards are there in the home page)

ayan-joshi commented:
> I told to make it in the home page itself instead of shifting to a new page (like services section cards are there in the home page)

ok so I need to make a section so I have to eliminate the cheasheet.html file ?

ayan-joshi committed:
commit

ayan-joshi commented:
@agamjotsingh18 I'm sure you'll like my work this time ,
Now I edited the navbar with a new cheatsheet section in the index file only which contains cheatsheets of 7 languages 
also I eliminated the cheatsheet.html file as their is no need of it now ,
Also I myself added all the cheatsheets pdf and added the links 
before only 3 cheatsheets were their which also I deleted as it's not needed now 
Have a look at it and please merge the request !

agamjotsingh18 commented:
Hey, just shift this section before testimonials section, also it's not looking good in dark theme...

Just a suggestion, Please look after these changes next time when contributing to open source, this increases the probability of your pr to be merged.

ayan-joshi committed:
commit

ayan-joshi commented:
> Hey, just shift this section before testimonials section, also it's not looking good in dark theme...
> 
> Just a suggestion, Please look after these changes next time when contributing to open source, this increases the probability of your pr to be merged.

ayan-joshi commented:
@agamjotsingh18 did changes u asked now it's fine with darkmode as well also added some css to it 
in my issue I had work to just add cheatsheet to main so I didn't looked after it css , just as I didn't found the content so I added the pdf of every cheatsheet to make it better

ayan-joshi commented:
@agamjotsingh18 please merge it

ayan-joshi commented:
@agamjotsingh18 please merge the request
